packages/app/src/specs/InlineSpecListTree.vue

```diff
@@ -124,7 +124,16 @@ const toggle = (row: UseCollapsibleTreeNode<SpecTreeNode<FuzzyFoundSpec>>, idx:
   row.toggle()
 }
 
-const submit = (row: UseCollapsibleTreeNode<SpecTreeNode<FuzzyFoundSpec>>, idx: number) => {
+const submitOrToggle = (row: UseCollapsibleTreeNode<SpecTreeNode<FuzzyFoundSpec>>, idx: number, event?: KeyboardEvent) => {
```

Comment:
Renamed this since it has condition logic that doesn't always "submit" the spec.